scripts/lib/CIME/test_scheduler.py

```diff
@@ -921,6 +921,15 @@ def run_tests(self, wait=False,
 
         expect(threading.active_count() == 1, "Leftover threads?")
 
+        # Copy TestStatus files to baselines for tests that have already failed.
```

Comment:
For my own edification: When does this code get invoked in the whole test process?

Reply:
At the end of create_test. So, on a batch machine, once all the tests are built and submitted.